app/models/asset_field_type.rb

```diff
@@ -52,6 +52,15 @@ def mapping
 
   private
 
+  def is_image?(mime_type)
+    ['image/jpeg', 'image/pjpeg', 'image/png','application/pdf' ,'image/x-png', 'image/gif'].include?(mime_type)
+  end
+
+  def format_data(metadata_hash)
```

Comment:
Please come up with a better name for this method - it isn't just formatting data